manual/manual/cmds/intf-c.etex

```diff
@@ -1535,12 +1535,15 @@ section~\ref{s:main-c}, this means that "argv[0]" or "argv[1]" must
 contain the executable file name.
 
 An alternative is to embed the bytecode in the C code. The
-"-output-obj" option to "ocamlc" is provided for this purpose.  It
-causes the "ocamlc" compiler to output a C object file (".o" file,
-".obj" under Windows) containing the bytecode for the OCaml part of the
-program, as well as a "caml_startup" function. The C object file
-produced by "ocamlc -output-obj" can then be linked with C code using
-the standard C compiler, or stored in a C library.
+"-output-obj" and "-output-complete-obj" options to "ocamlc" are
+provided for this purpose. It causes the "ocamlc" compiler to output a
+C object file (".o" file, ".obj" under Windows) containing the
+bytecode for the OCaml part of the program, as well as a
+"caml_startup" function. The C object file produced by "ocamlc
+-output-complete-obj" also contains the runtime and autolink
```

Comment:
I'm not sure if the larger context defines the notion of `autolink` libraries. But I guess that's the C linking instructions that are embedded in ocaml object's files.